server/UserRegistration.java

```diff
@@ -0,0 +1,467 @@
+                    isItAFloat = false;
+                    System.out.println("\nThis is not a correct value input");
+                }
+
+            } while (!isItAFloat);
+
+            do {
+                System.out.print("\nIs this correct (Y/N)?: ");
+                response = sc.nextLine().toUpperCase();
+                if (response.equals("N")) {
+                    System.out.print("Ok,what would you like the update opening balance to be?: ");
+                    userOpenBalance = sc.nextLine();
+                    try {
+                        float newBalance = Float.parseFloat(userOpenBalance);
+                        System.out.println("New Account Balance:" + "£" + newBalance);
+                    } catch (Exception e) {
+                        isItAFloat = false;
+                        System.out.println("\nThis is not a correct value input");
+                    }
+                    System.out.print("\nIs this correct (Y/N)?: ");
+                    response = sc.nextLine().toUpperCase();
+                }
+            } while (!response.equals("Y"));
+            System.out.println("New Account Balance:" + "£" + userOpenBalance);
+            Integer.valueOf(userOpenBalance);
+            return userOpenBalance;
+        }
+
+    }
+
+    private static class newUserSecretQuestion {
+        // set conditional
+        String quesAns;
+        Scanner sc = new Scanner(System.in);
+        String response;
+
+        public String secretQuestion() {
+            System.out.println(" ");
+            System.out.println("\nPlease now answer the following secret question");
+            System.out.println("\nWhat was your first pets name?: ");
+            quesAns = sc.nextLine();
+            System.out.println("The secret answer is set to: " + " " + quesAns);
+
+
+            do {
+                System.out.print("\nIs this correct (Y/N)?: ");
+                response = sc.nextLine().toUpperCase();
+                if (response.equals("N")) {
+                    System.out.print("Ok, you would like to change the secret answer to?: ");
+                    quesAns = sc.nextLine();
+                    System.out.print("\nThe secret answer associated with this account will be:" + " " + quesAns);
+                    System.out.print("\nIs this correct (Y/N)?: ");
+                    response = sc.nextLine().toUpperCase();
+                }
+            } while (!response.equals("Y"));
+
+            return quesAns;
+        }
+    }
+
+    private static class checkNotARobot {
+        Integer answer;
+        Integer userAnswer;
+        Scanner sc = new Scanner(System.in);
+
+
+        public boolean checkHuman() {
+            //No condition or iterations as checks once to ensure the user is human.
+            answer = 30;
+            System.out.println("\nPlease now complete the following question");
+            System.out.println("\nWhat is 5 x 6 = "); // Answer is 30.
```

Comment:
Idea: Instead of hardcoding 5 and 6, maybe get two random numbers from 1 to 10, calculate the result, and then compare the result with what the user has entered, if it's the same, it's not a robot :)